docs/connecting-external-llm-endpoint/azure-openai.md

```diff
@@ -0,0 +1,58 @@
+---
+icon: material/microsoft-azure
+---
+
+# Connecting External LLM Endpoint: <nobr>:material-microsoft-azure: Azure OpenAI</nobr>
+
+Kolena connects with [Azure OpenAI](https://azure.microsoft.com/en-us/products/ai-services/openai-service)
+to leverage your hosted LLM endpoint to do data enrichment and result evaluation.
+In this tutorial, we'll learn how to establish an integration between Kolena and your Azure OpenAI deployment.
+
+To get started, ensure you have administrator access within Kolena.
+Navigate to the "Integrations" tab on the [:kolena-organization-16: Organization Settings](https://app.kolena.com/redirect/organization?tab=integrations)
+page under the "LLM Integration" section and click "Add Integration", then "Azure OpenAI".
+
+Note: By default, Kolena will make up to 100 concurrent request to your Azure OpenAI deployment to speed things up,
+please make sure your deployment can handle this load.
+
+### Step 1: Create Azure LLM Registration for Kolena
+
+We will generate an App registration for Kolena in Azure and then assign roles to this registration:
+
+1. From the [Azure portal](https://portal.azure.com/#home), search for "App registrations" and navigate to this page
+1. Click "New Registration"
+    1. Under "Supported account types", select "Accounts in any organizational directory"
+    1. Click "Register" to save the App registration
+1. Click on the App registration you have created
+1. Note the "Tenant ID" and "Application (client) ID"
+1. Click "Certificates & secrets", then "New client secret"
+    1. Click "Add" to save this secret and note the key value
```

Comment:
nit: i think markdown will render the ordered list properly, but keeping the literal order in the code level is a good practice considering we have other instances to follow it already

Reply:
will do